src/editor/map/context_manager.cpp

```diff
@@ -190,9 +190,14 @@ void context_manager::load_map_dialog(bool force_same_context /* = false */)
 	if (fn.empty()) {
 		fn = default_dir_;
 	}
-	int res = dialogs::show_file_chooser_dialog(gui_.video(), fn, _("Choose a File to Open"));
-	if (res == 0) {
-		load_map(fn, !force_same_context);
+
+	gui2::tfile_dialog dlg;
+
+	dlg.set_title(_("Choose a File to Open"))
```

Comment:
'Select File to Open', perhaps? Similarly, on could also use Select in place of 'Choose a' or 'Choose' everywhere, but that's up to you. I don't really think the 'a' is necessary in either case, especially since its use is not consistent.